### PR TITLE
fix yaml daemonset

### PR DESCRIPTION
**Pull request by r0binak**
fix duplicate in daemonset

### CODE REVIEW

**Comment by nyrahul** [comment]
Hey @r0binak , many thanks for looking into this.
However, the deployment files are auto-generated from `deployments` folder. You run `make` in `deployments` folder, that will overwrite the `generic/kubearmor.yaml`.
The changes have to go in [here](https://github.com/kubearmor/KubeArmor/blob/main/deployments/get/defaults.go).

**Review by daemon1024 — APPROVED** [review]

**Comment by nyrahul** [comment]
Hey @r0binak, I took the liberty to update your commit with the required changes. Also added your sign-off in the DCO. Please check if it is ok. Will wait for your LGTM to merge.

**Comment by r0binak** [comment]
ok, when its finally merge?

**Pull request merged**
